compiler/test-resources/scripting/classpathReport.sc

```diff
@@ -1,4 +1,4 @@
-#!dist/target/pack/bin/scala -classpath dist/target/pack/lib/*
+#!/usr/bin/env -S bin/scala -classpath 'dist/target/pack/lib/*'
```

Comment:
https://unix.stackexchange.com/a/477651 - probably the only way to pass multiple args with splitting at white spaces